common/tvm_utility/tvm_utility-extras.cmake

```diff
@@ -70,15 +69,17 @@ function(get_neural_network MODEL_NAME MODEL_BACKEND DEPENDENCY)
     set(DOWNLOAD_DIR "${DATA_PATH}/downloads")
     set(SOURCE_DIR "${DATA_PATH}/models/${MODEL_NAME}")
     set(INSTALL_DIRECTORY "${DATA_PATH}/models/${MODEL_NAME}")
+    file(DOWNLOAD ${URL} "${DOWNLOAD_DIR}/${ARCHIVE_NAME}")
```

Comment:
That's a bit shaky. It works fine for the first build: it downloads from the bucket. But for the second build the archive will exist, so what I guess happens is that `URL` becomes the name of the archive (from line 60) and then cmake downloads the file into itself. On my computer this results in the `.tar.gz` file having a size of 0 on disk. But because the files had already been extracted on the first build then it still works. I guess it will break if going back and forth between target versions.
Maybe remove the first `if` case at line 60, and instead do the download conditionally here? I don't know if it will solve everything though.

Reply:
I'm taking days off this week. Will come back and fix it next week.